src/test/java/ebl/v2/ShippingInstructionIT.java

```diff
@@ -24,10 +26,14 @@
       loadFileAsString("ValidShippingInstruction.json");
 
   private final ObjectMapper objectMapper = new ObjectMapper();
+  private static ShippingInstructionTO shippingInstructionTO;
 
   @BeforeAll
   static void configs() throws IOException {
     TestConfig.init();
+
+    shippingInstructionTO = new ShippingInstructionTO();
```

Comment:
I would prefer these to go under `@BeforeEach` so that it refreshes the data in case it's used in multiple test cases.